android/gltfio-android/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,2 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Missing copyright header

Reply:
None of our existing manifests have copyright headers.

Reply:
They should :)

Reply:
Done.